common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -52,8 +53,11 @@
     public static final String YYYY_MM_DD_HH_MM_SS = "yyyy-MM-dd HH:mm:ss";
     public static final String YYYY_MM_DD_HH_MM_SS_SSS = "yyyy-MM-dd#HH:mm:ss:SSS";
     public static final String YYYYMMDDHHMMSS = "yyyyMMddHHmmss";
+    public static final int MAPPED_FILENAME_DIGITS = 20;
+    public static final Pattern MAPPED_FILENAME_PATTERN = Pattern.compile("^\\d{20}$");
```

Comment:
move it to MappedFileQueue and mark as private.

Reply:
Moved MAPPED_FILENAME_PATTERN to MappedfileQueue. The MAPPED_FILENAME_DIGITS remains in this file as it is reference in it.